src/routes/login/LoginView.vue

```diff
@@ -272,7 +272,11 @@ export default {
       this.username = '';
       this.pw = '';
       this.loading = false;
-      this.$router.push('/');
+      const {redirectId} = this.$route.params;
+      if (!redirectId) {
+        return this.$router.push('/');
+      }
+      return this.$router.push(`/lobby/${redirectId}`);
```

Comment:
I think this is fine for now, but it's notably limiting in that it assumes we only go to the lobby. We may later want to generalize the redirect but until we have a clear vision for the UX across pages I think this is solid